itou/templates/dashboard/edit_user_email.html

```diff
@@ -0,0 +1,26 @@
+{% extends "layout/content_small.html" %}
+{% load i18n %}
+{% load bootstrap4 %}
+
+{% block title %}{% translate "Modifier votre profil" %}{{ block.super }}{% endblock %}
+
+{% block content %}
+<h1>{% translate "Modifier votre adresse e-mail" %}</h1>
+
+<div class="alert alert-warning" role="alert">
+    {% blocktranslate %}
+        Ce changement entraînera une <b>déconnexion</b>.<br>
+        Veuillez alors vous reconnecter avec votre nouvelle adresse e-mail.
+    {% endblocktranslate %}
+</div>
+
+<form method="post" action="" class="js-prevent-multiple-submit js-submit-with-typeform">
+    {% csrf_token %}
+
+    {% bootstrap_form_errors form %}
+    {% bootstrap_form form %}
+
+    <button type="submit" class="btn btn-primary">{% translate "Enregistrer" %}</button>
+</form>
+
+{% endblock %}
```

Comment:
![m25Chf1i](https://user-images.githubusercontent.com/281139/101472952-89aacc80-3949-11eb-9a8a-20553c2c3440.jpg) _No newline at end of file_

Reply:
![image](https://user-images.githubusercontent.com/6150920/101506315-84fc0d80-3975-11eb-9278-d5d1a002c3f6.png)